frontend/src/router/index.ts

```diff
@@ -21,42 +22,63 @@ import SummaryView from '@/views/SummaryView.vue';
 // 3. (Not recommended) Within router below, use route-level code-splitting which generates a separately loaded javascript file for this route. Syntax: component: () => import(../components/<component>.vue) syntax.
 //    This fixes the issue, but seems to break using shared state (e.g. in ApplicationService).
 
+const breadcrumbRouteItems = {
+    dashboard: {
+        to: '/dashboard',
+        label: 'Manage Permissions',
+    },
+    addUserPermission: {
+        to: '/grant',
+        label: 'Add User Permission'
+    }
+} as Breadcrumb;
+
 const routes = [
     {
         path: '/',
         name: 'landing',
         meta: {
             title: 'Welcome to FAM',
-            layout: 'SimpleLayout'
+            layout: 'SimpleLayout',
+            hasBreadcrumb: false
         },
         component: LandingView,
     },
     {
-        path: '/dashboard',
-        name: 'Manage Permissions',
+        path: breadcrumbRouteItems.dashboard.to,
+        name: breadcrumbRouteItems.dashboard.label,
         meta: {
-            title: 'Manage Permissions',
-            layout: 'ProtectedLayout'
+            title: breadcrumbRouteItems.dashboard.label,
+            layout: 'ProtectedLayout',
+            hasBreadcrumb: false
         },
         component: ManagePermissionsView,
     },
     {
-        path: '/grant',
-        name: 'grant',
+        path: breadcrumbRouteItems.addUserPermission.to,
+        name: breadcrumbRouteItems.addUserPermission.label,
         meta: {
-            title: 'Grant Access',
-            layout: 'ProtectedLayout'
+            title: breadcrumbRouteItems.addUserPermission.label,
+            layout: 'ProtectedLayout',
+            hasBreadcrumb: true
         },
         component: GrantAccessView,
+        beforeEnter: () => {
+            populateBreadcrumb([breadcrumbRouteItems.dashboard])
```

Comment:
Hi @J0taFerreira , this is probably a design question.
Normally, for breadcrumb, should it be showing "**current navigation step**" for the user? Or is it the "last" navigation step?

Reply:
Hello! As per design it is showing only the last navigation to follow pattern of the other teams.

Reply:
Sure, if that's the design they decide. 
Normally I see the current step is the last item in the breadcrumb items order (with no link for that item), maybe that's been discussed.